src/plugins/intel_gpu/src/graph/impls/ocl/fully_connected.cpp

```diff
@@ -118,6 +118,9 @@ struct fully_connected_impl : typed_primitive_impl_ocl<fully_connected> {
             params.quantization = kernel_selector::QuantizationType::NONE;
         }
 
+        if (impl_param.get_program().get_node(primitive->id).is_dynamic())
+            params.is_dynamic = true;
```

Comment:
similar flag(`is_shape_agnostic`) has already been added on this PR(https://github.com/openvinotoolkit/openvino/pull/15317) for the same requirement. Please rebase and update to use it from base kernel params

Reply:
Updated, thank you